diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index 45f07db..2af6c9a 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -75,22 +75,22 @@ class OrderController extends AbstractController
 
         $products = $cart->getProducts();
         $variantId = $products[0]->getLsVariantId();
-        $quantity = $cart->getProductQuantity($products[0]);
-
-        $attributes = [
-            'checkout_data' => [
-                'variant_quantities' => [
-                    [
-                        'variant_id' => (int) $variantId,
-                        'quantity' => $quantity,
-                    ],
-                ],
-            ],
-        ];
+
+        $attributes = [];
         if ($user) {
             $attributes['checkout_data']['email'] = $user->getEmail();
             $attributes['checkout_data']['name'] = $user->getFirstName();
         }
+        if (count($products) === 1) {
+            $attributes['checkout_data']['variant_quantities'] = [
+                [
+                    'variant_id' => (int) $variantId,
+                    'quantity' => $cart->getProductQuantity($products[0]),
+                ],
+            ];
+        } else {
+
+        }
 
         $response = $lsClient->request(Request::METHOD_POST, 'checkouts', [
             'json' => [
