diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index cf9cb5b..7d5ed0c 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -81,6 +81,12 @@ class OrderController extends AbstractController
                                 'id' => '132127',
                             ],
                         ],
+                        'variant' => [
+                            'data' => [
+                                'type' => 'variants',
+                                'id' => '721878',
+                            ],
+                        ],
                     ],
                 ],
             ],
