diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index 574254f..33a8d98 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -35,6 +35,7 @@ final readonly class LemonSqueezyApi
         $attributes['checkout_data']['email'] = $user->getEmail();
         $attributes['checkout_data']['name'] = $user->getFirstName();
         $attributes['checkout_data']['custom']['user_id'] = (string) $user->getId();
+        $attributes['checkout_options']['embed'] = true;
         if (count($products) === 1) {
             $attributes['checkout_data']['variant_quantities'] = [
                 [
