diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index e77cdb9..f6295fe 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -77,6 +77,7 @@ class OrderController extends AbstractController
                 ],
             ],
         ]);
+        dd($response->getContent());
         $lsCheckout = $response->toArray();
 
         return $lsCheckout['data']['attributes']['url'];
