diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index 2798a33..72dfb57 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -57,6 +57,26 @@ class OrderController extends AbstractController
         HttpClientInterface $lsClient,
         ShoppingCart $cart,
     ): Response {
+        $lsCheckoutUrl = $this->createLsCheckoutUrl($lsClient, $cart);
 
+        return $this->redirect($lsCheckoutUrl);
+    }
+
+    private function createLsCheckoutUrl(HttpClientInterface $lsClient, ShoppingCart $cart): string
+    {
+        if ($cart->isEmpty()) {
+            throw new \LogicException('Nothing to checkout!');
+        }
+
+        $response = $lsClient->request(Request::METHOD_POST, 'checkouts', [
+            'json' => [
+                'data' => [
+                    'type' => 'checkouts',
+                ],
+            ],
+        ]);
+        $lsCheckout = $response->toArray();
+
+        return $lsCheckout['data']['attributes']['url'];
     }
 }
