diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index 7d5ed0c..6497358 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -91,7 +91,7 @@ class OrderController extends AbstractController
                 ],
             ],
         ]);
-        dd($response->getContent(false));
+        //dd($response->getContent(false));
         $lsCheckout = $response->toArray();
 
         return $lsCheckout['data']['attributes']['url'];
