diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index d8f0c6e..bb840d4 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -70,6 +70,7 @@ class OrderController extends AbstractController
     #[Route('/checkout/success', name: 'app_order_success')]
     public function success(
         Request $request,
+        ShoppingCart $cart,
     ): Response
     {
         $referer = $request->headers->get('referer');
