diff --git a/src/Entity/Product.php b/src/Entity/Product.php
index e1d88bd..c61e5e5 100644
--- a/src/Entity/Product.php
+++ b/src/Entity/Product.php
@@ -29,6 +29,9 @@ class Product
     #[ORM\Column(length: 255)]
     private ?string $imageFilename = null;
 
+    #[ORM\Column(length: 255, nullable: true)]
+    private ?string $lsVariantId = null;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -93,4 +96,16 @@ class Product
 
         return $this;
     }
+
+    public function getLsVariantId(): ?string
+    {
+        return $this->lsVariantId;
+    }
+
+    public function setLsVariantId(?string $lsVariantId): static
+    {
+        $this->lsVariantId = $lsVariantId;
+
+        return $this;
+    }
 }
