diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index a9ad48e..d09cc34 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -133,6 +133,28 @@ final readonly class LemonSqueezyApi
         } catch (ClientException $e) {
             $data = $e->getResponse()->toArray(false);
             //dd($data);
+
+            $mainErrorMessage = 'LS API Error:';
+
+            $error = $data['errors'][0] ?? null;
+            if ($error) {
+                if (isset($error['status'])) {
+                    $mainErrorMessage .= ' ' . $error['status'];
+                }
+                if (isset($error['title'])) {
+                    $mainErrorMessage .= ' ' . $error['title'];
+                }
+                if (isset($error['detail'])) {
+                    $mainErrorMessage .= ' "' . $error['detail'] . '"';
+                }
+                if (isset($error['source']['pointer'])) {
+                    $mainErrorMessage .= sprintf(' (at path "%s")', $error['source']['pointer']);
+                }
+            } else {
+                $mainErrorMessage .= $e->getResponse()->getContent(false);
+            }
+
+            throw new \Exception($mainErrorMessage, 0, $e);
         }
 
         return $data;
