diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index 6ceb42e..ab91c41 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -5,6 +5,7 @@ namespace App\Store;
 use App\Entity\User;
 use Symfony\Component\DependencyInjection\Attribute\Autowire;
 use Symfony\Component\DependencyInjection\Attribute\Target;
+use Symfony\Component\HttpClient\Exception\ClientException;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
@@ -126,5 +127,13 @@ final readonly class LemonSqueezyApi
 
     private function request(string $method, string $url, array $options = []): array
     {
+        try {
+            $response = $this->client->request($method, $url, $options);
+            $data = $response->toArray();
+        } catch (ClientException $e) {
+
+        }
+
+        return $data;
     }
 }
