diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index d09cc34..ab2fe65 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -34,7 +34,7 @@ final readonly class LemonSqueezyApi
         $attributes = [];
         $attributes['checkout_data']['email'] = $user->getEmail();
         $attributes['checkout_data']['name'] = $user->getFirstName();
-        $attributes['checkout_data']['custom']['user_id'] = $user->getId();
+        $attributes['checkout_data']['custom']['user_id'] = (string) $user->getId();
         if (count($products) === 1) {
             $attributes['checkout_data']['variant_quantities'] = [
                 [
