diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index ab2fe65..ca621fd 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -59,7 +59,7 @@ final readonly class LemonSqueezyApi
 
         $attributes['product_options']['redirect_url'] = $this->urlGenerator->generate('app_order_success', [], UrlGeneratorInterface::ABSOLUTE_URL);
 
-        $response = $this->request(Request::METHOD_POST, 'checkouts', [
+        $lsCheckout = $this->request(Request::METHOD_POST, 'checkouts', [
             'json' => [
                 'data' => [
                     'type' => 'checkouts',
@@ -81,8 +81,6 @@ final readonly class LemonSqueezyApi
                 ],
             ],
         ]);
-        dd($response->getContent(false));
-        $lsCheckout = $response->toArray();
 
         return $lsCheckout['data']['attributes']['url'];
     }
