diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index 98a189b..9ac2bf3 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -6,6 +6,7 @@ use App\Entity\Product;
 use App\Entity\User;
 use App\Store\LemonSqueezyApi;
 use App\Store\ShoppingCart;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\DependencyInjection\Attribute\Target;
 use Symfony\Component\HttpFoundation\Request;
@@ -80,6 +81,7 @@ class OrderController extends AbstractController
     #[Route('/checkout/handle', name: 'app_order_checkout_handle', methods: ['POST'])]
     public function handleCheckout(
         Request $request,
+        EntityManagerInterface $entityManager,
         #[CurrentUser] User $user,
     ): Response {
         $lsCustomerId = $request->request->get('lsCustomerId');
