diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index 9ac2bf3..146c1e6 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -86,6 +86,10 @@ class OrderController extends AbstractController
     ): Response {
         $lsCustomerId = $request->request->get('lsCustomerId');
         $user->setLsCustomerId($lsCustomerId);
+
+        $entityManager->flush();
+
+        return $this->json([]);
     }
 
     #[Route('/checkout/success', name: 'app_order_success')]
