diff --git a/assets/controllers/lemon-squeezy_controller.js b/assets/controllers/lemon-squeezy_controller.js
index 4ed9b3e..713cfe5 100644
--- a/assets/controllers/lemon-squeezy_controller.js
+++ b/assets/controllers/lemon-squeezy_controller.js
@@ -66,6 +66,28 @@ export default class extends Controller {
     }
 
     #handleCheckout(lsCustomerId) {
+        fetch(this.checkoutHandleUrlValue, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded',
+            },
+            body: new URLSearchParams({
+                lsCustomerId: lsCustomerId,
+            }),
+        })
+            .then(response => {
+                if (!response.ok) {
+                    throw new Error("Network response was not ok " + response.statusText);
+                }
+
+                return response.json();
+            })
+            .then(data => {
+                // Nothing to do
+            })
+            .catch(error => {
+                console.error('Fetch error:', error);
+            });
     }
 
     #disableLink(link) {
