diff --git a/src/Controller/OrderController.php b/src/Controller/OrderController.php
index 146c1e6..81a6137 100644
--- a/src/Controller/OrderController.php
+++ b/src/Controller/OrderController.php
@@ -84,6 +84,11 @@ class OrderController extends AbstractController
         EntityManagerInterface $entityManager,
         #[CurrentUser] User $user,
     ): Response {
+        $userId = $request->request->get('userId');
+        if ($userId !== (string) $user->getId()) {
+            throw $this->createAccessDeniedException(sprintf('Current user ID "%s" does not match the user ID "%s" of the order!', $user->getId(), $userId));
+        }
+
         $lsCustomerId = $request->request->get('lsCustomerId');
         $user->setLsCustomerId($lsCustomerId);
 
