diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index 5451799..8595ab2 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -96,11 +96,13 @@ final readonly class LemonSqueezyApi
 
     public function listOrders(User $user): array
     {
+        $userEmail = $user->getEmail();
+
         $response = $this->client->request(Request::METHOD_GET, 'orders', [
             'query' => [
                 'filter' => [
                     'store_id' => $this->storeId,
-                    'user_email' => $user->getEmail(),
+                    'user_email' => $userEmail,
                 ],
                 'page' => [
                     'size' => 5, // @see https://docs.lemonsqueezy.com/api/getting-started/requests#pagination
