diff --git a/tests/Webhook/LemonSqueezyRequestParserTest.php b/tests/Webhook/LemonSqueezyRequestParserTest.php
index 8efdc5e..f975320 100644
--- a/tests/Webhook/LemonSqueezyRequestParserTest.php
+++ b/tests/Webhook/LemonSqueezyRequestParserTest.php
@@ -21,7 +21,10 @@ class LemonSqueezyRequestParserTest extends WebTestCase
         ]);
 
         $json = file_get_contents(__DIR__.'/../fixtures/order_created.json');
-        $client->request('POST', '/webhook/lemon-squeezy', [], [], [], $json);
+        $hash = hash_hmac('sha256', $json, $_ENV['LEMON_SQUEEZY_SIGNING_SECRET']);
+        $client->request('POST', '/webhook/lemon-squeezy', [], [], [
+            'HTTP_X-Signature' => $hash,
+        ], $json);
 
         self::assertResponseIsSuccessful('Webhook failed!');
         self::assertNotNull($user->getLsCustomerId(), 'LemonSqueezy customer ID not set!');
