diff --git a/tests/Webhook/LemonSqueezyRequestParserTest.php b/tests/Webhook/LemonSqueezyRequestParserTest.php
new file mode 100644
index 0000000..afd18d2
--- /dev/null
+++ b/tests/Webhook/LemonSqueezyRequestParserTest.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Tests\Webhook;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class LemonSqueezyRequestParserTest extends WebTestCase
+{
+    public function testSomething(): void
+    {
+        $client = static::createClient();
+        $crawler = $client->request('GET', '/');
+
+        $this->assertResponseIsSuccessful();
+        $this->assertSelectorTextContains('h1', 'Hello World');
+    }
+}
