diff --git a/src/Webhook/LemonSqueezyRequestParser.php b/src/Webhook/LemonSqueezyRequestParser.php
index 1d1cbb9..cb2fbe8 100644
--- a/src/Webhook/LemonSqueezyRequestParser.php
+++ b/src/Webhook/LemonSqueezyRequestParser.php
@@ -38,6 +38,9 @@ final class LemonSqueezyRequestParser extends AbstractRequestParser
         if (!$eventName || !$webhookId) {
             throw new RejectWebhookException(Response::HTTP_BAD_REQUEST, 'Request payload does not contain required fields.');
         }
+        if ($eventName !== 'order_created') {
+            throw new RejectWebhookException(Response::HTTP_BAD_REQUEST, sprintf('Unsupported event type: %s', $eventName));
+        }
     }
 
     private function verifySignature(Request $request, string $secret): void
